src/master-playlist-controller.js

```diff
@@ -820,14 +820,13 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     if (!buffered.length) {
       // return true if the playhead reached the absolute end of the playlist
-      return absolutePlaylistEnd - currentTime <= Ranges.TIME_FUDGE_FACTOR;
+      return absolutePlaylistEnd - currentTime <= 0.1;
     }
     let bufferedEnd = buffered.end(buffered.length - 1);
 
     // return true if there is too little buffer left and
     // buffer has reached absolute end of playlist
-    return bufferedEnd - currentTime <= Ranges.TIME_FUDGE_FACTOR &&
-           absolutePlaylistEnd - bufferedEnd <= Ranges.TIME_FUDGE_FACTOR;
+    return bufferedEnd - currentTime <= 0.1 && absolutePlaylistEnd - bufferedEnd <= 0.1;
```

Comment:
Same re: comment. Or we can have a constant elsewhere. Like `Ranges.SAFE_END_DELTA` and leave a comment above it.